org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/extensions/xsd/contentmodel/CMXSDAttributeDeclaration.java

```diff
@@ -35,12 +38,16 @@
 public class CMXSDAttributeDeclaration implements CMAttributeDeclaration {
 
 	private final XSAttributeUse attributeUse;
-	private String documentation;
+
+	private String tagDocumentation;
 
 	private Map<String, String> valuesDocumentation;

```

Comment:
final

Reply:
Currently, I cannot set to final because it gets reassigned here: 

https://github.com/eclipse/lemminx/blob/1080bf5860753537aea27a612e486dc35b0041f1/org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/extensions/xsd/contentmodel/CMXSDAttributeDeclaration.java#L106-L109

Reply:
replace with 

`this.valuesDocumentation .clear();`